src/ngx_http_lua_directive.c

```diff
@@ -450,6 +450,8 @@ ngx_http_lua_rewrite_by_lua(ngx_conf_t *cf, ngx_command_t *cmd, void *conf)
     ngx_str_t                   *value;
     ngx_http_lua_main_conf_t    *lmcf;
     ngx_http_lua_loc_conf_t     *llcf = conf;
+    ngx_array_t                 *p_rewrites = llcf->rewrites;
```

Comment:
We usually do not use the `p_blahblah` naming convention. So better avoid that here to be more consistent.